apps/judicial-system/web/src/routes/Court/Indictments/Completed/Completed.tsx

```diff
@@ -147,139 +147,139 @@ const Completed: FC = () => {
-                                serviceRequirement:
-                                  ServiceRequirement.NOT_APPLICABLE,
-                              },
-                              setWorkingCase,
-                            )
-                          }}
-                          large
-                          backgroundColor="white"
-                          label={formatMessage(
-                            strings.serviceRequirementNotApplicable,
-                          )}
-                        />
-                      </Box>
-                      <Box marginBottom={2}>
-                        <RadioButton
-                          id={`defendant-${defendant.id}-service-requirement-required`}
-                          name={`defendant-${defendant.id}-service-requirement`}
-                          checked={
-                            defendant.serviceRequirement ===
-                            ServiceRequirement.REQUIRED
-                          }
-                          onChange={() => {
-                            setAndSendDefendantToServer(
-                              {
-                                defendantId: defendant.id,
-                                caseId: workingCase.id,
-                                serviceRequirement: ServiceRequirement.REQUIRED,
-                              },
-                              setWorkingCase,
-                            )
-                          }}
-                          large
-                          backgroundColor="white"
-                          label={formatMessage(
-                            strings.serviceRequirementRequired,
-                          )}
-                        />
-                      </Box>
-                      <RadioButton
-                        id={`defendant-${defendant.id}-service-requirement-not-required`}
-                        name={`defendant-${defendant.id}-service-requirement`}
-                        checked={
-                          defendant.serviceRequirement ===
-                          ServiceRequirement.NOT_REQUIRED
-                        }
-                        onChange={() => {
-                          setAndSendDefendantToServer(
-                            {
-                              defendantId: defendant.id,
-                              caseId: workingCase.id,
-                              serviceRequirement:
-                                ServiceRequirement.NOT_REQUIRED,
-                            },
-                            setWorkingCase,
-                          )
-                        }}
-                        large
-                        backgroundColor="white"
-                        label={formatMessage(
-                          strings.serviceRequirementNotRequired,
-                        )}
-                      />
-                    </BlueBox>
+        {!sentToPublicProsecutor && isRulingOrFine && (
```

Comment:
If I understand this correctly, this will hide the radio buttons when the case has been sent to the public prosecutor. Shouldn't `sentToPublicProsecutor` be removed from the condition?